common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -200,6 +200,15 @@ class CapaFields(object):
         scope=Scope.settings
     )
 
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
I think this could move up to `EditingDescriptor` (which is the base class used by all of these XModuleDescriptors that have `data` as a core value).